reify-splash/index.html

```diff
@@ -41,32 +41,36 @@
         <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
           <ul class="nav navbar-nav navbar-right">
             <li><a href="#">FAQ</a></li>
-            <li><a href="partners.html">Partners</a></li>
+            <li><a href="#">Partners</a></li>
+            <li><a href="#">Events</a></li>
           </ul>
         </div><!-- /.navbar-collapse -->
       <!-- /.container-fluid -->
     </nav>
   </div>
 
-      <!-- Header -->
-  <header class="header">
-        <div class="text-vertical-center">
-            <h1 id="reifyLogo">REIFY</h1>
-            <h3 id="tagLine">make your perfect fit a reality</h3>
-            <br>
-            <!--  This will be the login  -->
-            <p class="lead">
-              <div class="btn-group">
-                  <a class="btn btn-dark btn-lg" href="#">Login</a>
-                  <button type="button" class="btn btn-dark btn-lg" data-toggle="modal" data-target="#myModal">
-                   Sign Up!
-                  </button>
-              </div>
-            </p>
 
-            <!-- -->
+
+<div id="jumbopic" class="jumbotron">
+          <div class="container">
+
+              <div id ="jumbotron-registration" class="row">
+
+                  <div class="col-md-12 col-md-offset-2">
+                      <h1 id="reifyLogo">REIFY</h1>
+                      <p id="tagLine">make your perfect fit a reality.</p>
+                      <p class="lead">
```

Comment:
Make sure youre  using naming conventions in html with a dash "-" instead of camel case. ex. reify-logo and not reifyLogo